Shared/src/com/pacoapp/paco/shared/util/ExperimentHelper.java

```diff
@@ -236,8 +236,11 @@ public static boolean isAnyGroupOngoingDuration(ExperimentDAO experiment) {
     List<ExperimentGroup> listeningExperimentGroups  = new ArrayList();
     List<ExperimentGroup> experimentGroups = experiment.getGroups();
     for (ExperimentGroup experimentGroup : experimentGroups) {
-      if (experimentGroup.getBackgroundListen() && experimentGroup.getBackgroundListenSourceIdentifier().equals(sourceIdentifier)) {
-        listeningExperimentGroups.add(experimentGroup);
+      if (experimentGroup.getBackgroundListen()) {
```

Comment:
Are you using backgroundListen? This is the feature where another app on Android can send information under a sourceIdentifier into the Paco dataset for an experiment. 

I think you might have wanted it somewhere else as well. It is smart to make all sourceIdentifier matching codes use the wildcard for consistency sake, but, this may not be the place you needed.